packages/webview_flutter/example/lib/main.dart

```diff
@@ -20,14 +26,28 @@ class WebViewExample extends StatelessWidget {
         // This drop down menu demonstrates that Flutter widgets can be shown over the web view.
         actions: <Widget>[
           NavigationControls(_controller.future),
-          const SampleMenu(),
+          Builder(
+            builder: (BuildContext context) {
+              return SampleMenu(onSelected: (String value) {
```

Comment:
I see the value of keeping `SampleMenu` dumb and unaware of the webview, but in this case as the app is small and is all about webview I think it will be a littler easier to read and more consistent with the other controls to pass the `_controller`'s future to `SampleMenu` and use a `FutureBuilder` to only allow the `evaluateJavaScript` call when we have a controller.